app/components/CreateUser/CreateUser.js

```diff
@@ -11,16 +12,15 @@ export class CreateUser extends Component {
   }
 
   render() {
-    let { loadSignupSuccess, loadSignupErrored, userRegisterDetails } = this.props
+    let { loadSignupSuccess, userRegisterDetails } = this.props
 
     return (
       <div className="row">
         <div className="col-sm-push-2 col-sm-8 col-xs-12">
-          {loadSignupSuccess &&
-            <div>
-              <CreateUserForm initialState={userRegisterDetails} />
-            </div>}
-          {loadSignupErrored && <div>error loading user registration details</div>}
+          <article role="main">
```

Comment:
Why are you using article? Coding html like it is 1999 :P?

Reply:
I'm just copying what was already there :)

Should it be main?